spanner/benchmark/ycsb.py

```diff
@@ -0,0 +1,257 @@
+"""The YCSB client in Python.
+
+Usage:
+
+  # Set up instance and load data into database.
+
+  # Set up environment variables. You should use your own credentials and gcloud
+  # project.
+  $ export GOOGLE_APPLICATION_CREDENTIALS=/path/to/credentials.json
+  $ export GCLOUD_PROJECT=gcloud-project-name
+
+  # Run the benchmark.
+  $ python spanner/benchmark/ycsb.py run -P pkb/workloada -p table=usertable \
```

Comment:
should this be workload or is this right?

Reply:
YCSB has several predefined workload which are named as workload[a|b||...]. The most common ones are workloada and workloadb. Here I just use workloada.